diff -urNad jack-1.9.7-orig/linux/freebob/JackFreebobDriver.cpp jack-1.9.7/linux/freebob/JackFreebobDriver.cpp
--- jack-1.9.7-orig/linux/freebob/JackFreebobDriver.cpp	2011-03-30 18:04:28.000000000 +0300
+++ jack-1.9.7/linux/freebob/JackFreebobDriver.cpp	2011-08-07 12:51:19.000000000 +0300
@@ -768,7 +768,7 @@
             }
             port = fGraphManager->GetPort(port_index);
             // Add one buffer more latency if "async" mode is used...
-            range.min = range.max = (driver->period_size * (driver->device_options.nb_buffers - 1)) + ((fEngineControl->fSyncMode) ? 0 : fEngineControl->fBufferSize) + driver->playback_frame_latency);
+            range.min = range.max = (driver->period_size * (driver->device_options.nb_buffers - 1)) + ((fEngineControl->fSyncMode) ? 0 : fEngineControl->fBufferSize) + driver->playback_frame_latency;
             port->SetLatencyRange(JackPlaybackLatency, &range);
             fPlaybackPortList[i] = port_index;
             jack_log("JackFreebobDriver::Attach fPlaybackPortList[i] %ld ", port_index);
